idaes/gas_solid_contactors/unit_models/moving_bed.py

```diff
@@ -632,20 +665,33 @@ def gas_phase_config_pressure_drop(b, t, x):
                              doc="Gas side pressure drop calculation -"
                                  "ergun equation")
             def gas_phase_config_pressure_drop(b, t, x):
-                return (1e2*-b.gas_phase.deltaP[t, x]*1e5 ==
-                        1e2*(
-                        150*(1 - b.bed_voidage) ** 2 *
+                #  add 1/pyunits.s to account for missing parameter units
+                return (-pyunits.convert(
+                    b.gas_phase.deltaP[t, x],
+                    to_units=units_meta_solid('pressure') /
+                    units_meta_solid('length')) ==
+                        (
+                        150/pyunits.s*(1 - b.bed_voidage) ** 2 *
```

Comment:
As above, where does this equation come from?

Reply:
Similar to above.